### PR TITLE
Create MeiTuan.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/MeiTuan.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Mixed Content for map.qq.com :
+	city.*.com
+	hotel.meituan.com
+	
+-->
+
+<ruleset name="MeiTuan (partial)">
+
+	<target host="b.meituan.com"/>
```
Comment (J0WI) [inline src/chrome/content/rules/MeiTuan.xml]: bj.meituan.com

File: src/chrome/content/rules/MeiTuan.xml [hunk 2]
```diff
@@ -36,6 +36,28 @@ Mixed Content for map.qq.com :
 	<target host="s1.meituan.net"/>
 	<target host="xs01.meituan.net"/>
 	
+	<!--	Too much to list all cities : https://www.meituan.com/index/changecity/initiative	-->
```
Comment (J0WI) [inline src/chrome/content/rules/MeiTuan.xml]: You can easily grep them out of the source code.

Comment (renyouguo) [inline src/chrome/content/rules/MeiTuan.xml]: @J0WI Too much to over test.

**Comment by J0WI** [comment]
This are to much hosts for Travis, I test them locally.

**Comment by J0WI** [comment]
```
ERROR rules/MeiTuan.xml: Fetch error: http://www.meituan.com/ => https://www.meituan.com/: (28, 'Operation timed out after 30000 milliseconds with 146412 bytes received')
ERROR rules/MeiTuan.xml: Fetch error: http://as.meituan.com/ => https://as.meituan.com/: (28, 'Operation timed out after 20000 milliseconds with 0 out of 0 bytes received')
ERROR rules/MeiTuan.xml: Fetch error: http://als.meituan.com/ => https://als.meituan.com/: (28, 'Operation timed out after 30000 milliseconds with 98043 bytes received')
ERROR rules/MeiTuan.xml: Fetch error: http://anqing.meituan.com/ => https://anqing.meituan.com/: (28, 'Operation timed out after 30001 milliseconds with 130810 bytes received')
ERROR rules/MeiTuan.xml: Fetch error: http://ay.meituan.com/ => https://ay.meituan.com/: (28, 'Operation timed out after 30000 milliseconds with 229113 bytes received')
ERROR rules/MeiTuan.xml: Fetch error: http://anshun.meituan.com/ => https://anshun.meituan.com/: (28, 'Operation timed out after 30000 milliseconds with 196347 bytes received')
ERROR rules/MeiTuan.xml: Fetch error: http://ankang.meituan.com/ => https://ankang.meituan.com/: (28, 'Operation timed out after 30001 milliseconds with 163579 bytes received')
ERROR rules/MeiTuan.xml: Fetch error: http://hj.meituan.com/ => https://hj.meituan.com/: (35, 'Unknown SSL protocol error in connection to hj.meituan.com:443 ')
```

**Comment by renyouguo** [comment]
@J0WI Sadly.
Would you please recheck it?
I doubt whether it will get the same error.

**Comment by J0WI** [comment]
They work when I test them separately.
I would like to wait until #5015 has been merged, there might be other hosts that time out next time.

**Comment by renyouguo** [comment]
F**k the GFW !

**Comment by J0WI** [comment]
@Hainish does https://github.com/EFForg/https-everywhere/pull/5626#issuecomment-235701835 also mean that this won't break the master? Because this rule LGTM.

**Comment by Hainish** [comment]
@J0WI it shouldn't break master.  If all the non-fetch tests work, and you perform the fetch test on your local (I suggest doing it the exact same way we specify in `.travis.yml`, with docker and all) and you don't see any problems, it should be fine to merge.

**Comment by J0WI** [comment]
Thanks for adding this!

**Comment by renyouguo** [comment]
@J0WI Thanks.
There are only 4 PRs by me waiting your check.

**Pull request merged**